tests/unittests/tests-gnrc_mac_internal/tests-gnrc_mac_internal.c

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright (C) 2016, 2016 Shuguo Zhuo <shuguo.zhuo@inria.fr>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ */
+#include <string.h>
+
+#include "embUnit.h"
+
+#include "net/gnrc/pkt.h"
+#include "net/gnrc/mac/internal.h"
+
+#include "unittests-constants.h"
+#include "tests-gnrc_mac_internal.h"
+
+static void set_up(void)
+{
+    gnrc_pktbuf_init();
+}
+
+#if GNRC_MAC_TX_QUEUE_SIZE != 0
+static void test_gnrc_mac_queue_tx_packet(void)
```

Comment:
Considering the length of this test I would prefer a more descriptive test-case name or some comment describing the test and the expected behavior.